02week/pigLatin.js

```diff
@@ -9,9 +9,45 @@ const rl = readline.createInterface({
 
 
 function pigLatin(word) {
+	//ex. car === arcay
 
-  // Your code here
+  //global variables
+	const wordArray = word.toLowerCase().trim().split('');
+	const vowels = ['a', 'e', 'i', 'o', 'u'];
 
+	//take the first letters of the word up to the first vowel
+		//firstLetters()
+		//comparing the word and vowels arrays to find the first instance of a vowel in the word, indexOf
+	const firstLetters = (word) => {
+		for(let i = 0; i < wordArray.length; i++){
+			for(let j = 0; j < vowels.length; j++){
```

Comment:
This second loop is unnecessary. Check for the existence of the letter in the vowel array using the same method as line 25.